mozilla/fx-private-relay
# MPP-3257: change send_first_email signal receiver to send_welcome_email command

This PR fixes #MPP-3257.

How to test:
1. Check out this branch
2. Run migrations
3. Sign up as a new Relay user
   * [ ] Relay should NOT send the welcome email immediately
4. Run the new `send_welcome_emails` command: `python manage.py send_welcome_emails`
   * [ ] Relay SHOULD send the welcome email
5. Set your browser to a non-English language supported by Relay
6. Register a new FxA user
7. Sign up for Relay as this non-English-language FxA user
   * [ ] Relay should NOT send the welcome email immediately
4. Run the new `send_welcome_emails` command: `python manage.py send_welcome_emails`
   * [ ] Relay SHOULD send the welcome email *IN THE APPROPRIATE LANGUAGE*
